test/print_test_stats.py

```diff
@@ -165,6 +194,11 @@ def positive_float(value):
         metavar="N",
         help="how many longest tests to show from the entire run",
     )
+    parser.add_argument(
+        "--upload-to-s3",
```

Comment:
i think we don't need this flag -
if we are uploading test statistics for future analysis, then it only make sense to upload master/release branch results? --> we might be able to get that from os env.

Reply:
I added the flag because I don't want the script to upload random data to S3 every time I run it locally, even if all I want to do is just get test stats info from some tests I ran locally